### ISSUE TITLE
Images don't have size property

### ISSUE THREAD

**Issue by miminar** [openshift/origin#16306]
The image size is not set on images pushed to recent (latest) registry/master.

See the image built with OpenShift builder:

```
oc describe image sha256:12e63f54fca507b2016d2746dd4747e9a5aa0587204a02efc6d25ba51a311e74
Docker Image:   172.30.30.30:5000/extended-test-limitrange-admission-bn8kz-splls/sized@sha256:12e63f54fca507b2016d2746dd4747e9a5aa0587204a02efc6d25ba51a311e74
Name:           sha256:12e63f54fca507b2016d2746dd4747e9a5aa0587204a02efc6d25ba51a311e74
Created:        About a minute ago
Annotations:    openshift.io/image.managed=true
Image Size:     0 B (first layer 15.09 kB, last binary layer 32 B)
Image Created:  About a minute ago
Author:         <none>
Arch:           amd64
Command:        <none>
Working Dir:    <none>
User:           <none>
Exposes Ports:  <none>
Docker Labels:  io.openshift.build.name=sized-1
                io.openshift.build.namespace=extended-test-limitrange-admission-bn8kz-splls
Environment:    PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
                OPENSHIFT_BUILD_NAME=sized-1
                OPENSHIFT_BUILD_NAMESPACE=extended-test-limitrange-admission-bn8kz-splls
```

Note the `Image Size`. Because of that, quota/limits don't work. And our extended tests are failing.

**Comment by pweil-** [comment]
@miminar please confirm whether or not this is a regression in 3.6 that was missed in testing.  If so let's use a BZ to ensure we get cases and backports

/cc @bparees

**Comment by miminar** [comment]
Yes, this is a regression from 3.6. Our unit and integration tests don't cover it. It's covered just by optional extended tests.

**Comment by bparees** [comment]
it's a regression since 3.6, it wasn't a regression the shipped(broker) in 3.6, right?

So our test coverage is ok, no backports needed, just need to fix it and continue to monitor our extended(optional) tests, for which we will start running a nightly job soon.

**Comment by miminar** [comment]
Fix available in #16325
